### PR TITLE
register pytest mark

### LINKED ISSUE #408: Register mark `spe1` or delete it
Tests in [`tests/data/spe1_st/tests/test_webviz_ert.py`](https://github.com/equinor/webviz-ert/tree/main/tests/data/spe1_st/tests) are correctly marked with `@pytest.mark.spe1` but this mark is not registered in `pyproject.toml` so `pytest` comes back with 

```python
tests/data/spe1_st/tests/test_webviz_ert.py:86
  /tmp/mtha/webviz-ert/tests/data/spe1_st/tests/test_webviz_ert.py:86: PytestUnknownMarkWarning: Unknown pytest.mark.spe1 - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
    @pytest.mark.spe1
```

Options: register the mark, remove the marked tests and just use `-k spe1`, or ignore it.

### PR DESCRIPTION
**Pull request by kwinkunks**
The mark is present in the tests but not registered. This commit registers it in `pyproject.toml` and uses it for the default test selection instead of -k.

**Issue**
Resolves #408 

**Approach**
Register the mark in the init file.

## Pre review checklist

- [ ] Added appropriate labels

### CODE REVIEW

**Comment by ertomatic** [comment]
Can one of the admins verify this patch?

File: pyproject.toml [hunk 1]
```diff
@@ -2,4 +2,7 @@
 requires = ["setuptools", "setuptools_scm", "wheel"]
 
 [tool.pytest.ini_options]
-addopts = "-k 'not spe1'"
+addopts = "-m 'not spe1'"
+markers = [
+    "spe1: marks tests needing results from ert forward model spe1",
```
Comment (berland) [inline pyproject.toml:7]: to be pedantic, maybe rephrase to "from ert model of the spe1 test case". 

(an ert model consists (usually) of multiple forward models, and spe1 is really an external thing from Society of Petroleum Engineers)

**Review by berland — APPROVED** [review]
